products/compute/api.yaml

```diff
@@ -562,6 +561,64 @@ objects:
             all responses from this backend had a "Cache-Control: public,
             max-age=[TTL]" header, regardless of any existing Cache-Control
             header. The actual headers served in responses will not be altered.
+        - !ruby/object:Api::Type::Integer
+          name: 'defaultTtl'
+          min_version: beta
+          description: |
+            Specifies the default TTL for cached content served by this origin for responses 
+            that do not have an existing valid TTL (max-age or s-max-age).
+        - !ruby/object:Api::Type::Integer
+          name: 'maxTtl'
+          min_version: beta
+          description: |
+            Specifies the maximum allowed TTL for cached content served by this origin.
+        - !ruby/object:Api::Type::Integer
+          name: 'clientTtl'
+          min_version: beta
+          description: |
+            Specifies the maximum allowed TTL for cached content served by this origin.
+        - !ruby/object:Api::Type::Boolean
+          name: 'negativeCaching'
+          min_version: beta
+          description: |
+            Negative caching allows per-status code TTLs to be set, in order to apply fine-grained caching for common errors or redirects.
+        - !ruby/object:Api::Type::Array
+          name: 'negativeCachingPolicy'
+          min_version: beta
+          description: |
+            Sets a cache TTL for the specified HTTP status code. negativeCaching must be enabled to configure negativeCachingPolicy.
+            Omitting the policy and leaving negativeCaching enabled will use Cloud CDN's default cache TTLs.
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::Integer
+                name: 'code'
```

Comment:
Is this or ttl required? Are neither?

Reply:
No of the fields are required by the API